src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/InfosetObjects/XamlObjectWriter.cs

```diff
@@ -782,6 +782,10 @@ public override void WriteEndMember()
                                 shouldSetValue = true;
                                 _context.ParentKeyIsUnconverted = true;
                             }
+                            else if (valueXamlType == XamlLanguage.String && property.Type.IsCollection && property.TypeConverter == null && property.Type.ItemType.TypeConverter != null)
```

Comment:
this would exclude collections of type string/object because they don't need a converter

Reply:
(not necessarily bad thing and can be enabled later)